components/Sections/ImageTextColumns.tsx

```diff
@@ -0,0 +1,59 @@
+import { getImageInfo } from "@/lib/getAssetInfo";
+import prisma, { Prisma } from "@/lib/prisma";
+import Image from "next/image";
+import React from "react";
+
+// ToDo: It crashes when the leftImage or rightImage is null
+
+export type ImageTextColumnsProps = Prisma.ImageTextColumnsGetPayload<{}>;
+
+export async function ImageTextColumns({
+  title,
+  content,
+  leftImage,
+  rightImage,
+}: ImageTextColumnsProps) {
+  const infoLeft = await getImageInfo(String(leftImage))
+    .then((res) => res)
+    .catch((err) => null);
```

Comment:
`.then` and `.catch` are useless in this case. The `await` already awaits the promise.

Reply:
The problem is that we not always get something back from it.

Reply:
The promise rejects when there is no image provided, but just check if there’s an image first before even calling the function.